src/ui/room_symbol.ui

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>room_symbol</class>
+ <widget class="QDialog" name="room_symbol">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>304</width>
+    <height>166</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Room symbol</string>
+  </property>
+  <layout class="QGridLayout" name="gridLayout">
```

Comment:
I think you have some redundancy here - this `QGridLayout` only has one member..

Reply:
Leftover after playing around with layout. Removed.